src/components/scenes/RequestScene.tsx

```diff
@@ -593,7 +603,8 @@ export const RequestScene = connect<StateProps, DispatchProps, OwnProps>(
         account,
         publicAddress: '',
         fioAddressesExist: false,
-        isConnected: state.network.isConnected
+        isConnected: state.network.isConnected,
+        isShowBalance: state.ui.settings.isAccountBalanceVisible
```

Comment:
Incorrect variable grammar. should just be `showBalance`

Reply:
I thought we only drop 'is' from component external-facing OwnProps? this seems inconsistent with isConnected and isAccountbAlanceVisible...